lms/djangoapps/grades/new/course_data.py

```diff
@@ -0,0 +1,89 @@
+from lms.djangoapps.course_blocks.api import get_course_blocks
+from xmodule.modulestore.django import modulestore
+from ..transformer import GradesTransformer
+
+
+class CourseData(object):
```

Comment:
Sure.  Good idea.  Or, we wait until another feature actually wants to make use of it.